public/index.html

```diff
@@ -1,11 +1,14 @@
 <!DOCTYPE html>
 <html lang="en">
-  <head>
-    <meta charset="utf-8" />
-    <meta name="viewport" content="width=device-width, initial-scale=1" />
-    <title>React Sum</title>
-  </head>
-  <body>
-    <div id="root"></div>
-  </body>
+
+<head>
+  <meta charset="utf-8" />
+  <meta name="viewport" content="width=device-width, initial-scale=1" />
+  <title>React Sum</title>
+</head>
+
+<body>
+  <div id="root"></div>
+</body>
+
```

Comment:
There was no need in changing this file, but if you already did it, let's do it proper way
```suggestion
	<head>
	    <meta charset="utf-8" />
	    <meta name="viewport" content="width=device-width, initial-scale=1" />
	    <title>React Sum</title>
	</head>

	<body>
  	<div id="root"></div>
	</body>
```